linux_os/guide/services/cron_and_at/restrict_at_cron_users/file_at_deny_exists/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
```

Comment:
`/etc/at.deny` does not need to exist for this recommendation so this rule isn't needed

Reply:
Removed